tdrs-frontend/src/components/UploadReport/UploadReport.test.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import thunk from 'redux-thunk'
+import { Provider } from 'react-redux'
+import configureStore from 'redux-mock-store'
+import { fireEvent, render } from '@testing-library/react'
+import axios from 'axios'
+
+import UploadReport from './UploadReport'
+
+describe('UploadReport', () => {
+  const initialState = {
+    reports: {
+      files: [
+        {
+          section: 'Active Case Data',
+          fileName: null,
+          error: null,
+        },
+        {
+          section: 'Closed Case Data',
+          fileName: null,
+          error: null,
+        },
+        {
+          section: 'Aggregate Data',
+          fileName: null,
+          error: null,
+        },
+        {
+          section: 'Stratum Data',
+          fileName: null,
+          error: null,
+        },
+      ],
+    },
+  }
+  const mockStore = configureStore([thunk])
+  const handleCancel = jest.fn()
+
+  it('should render four inputs for uploading files', () => {
```

Comment:
just noting for the future, I might expect for the # of inputs in the initial state to be tied to the STT and based on whether sample data (section 4) is reported.